tests/test_upload_n_download_excel.py

```diff
@@ -71,8 +76,11 @@ def test_no_file_type(self):
     def test_override_file_name(self):
         for file_type in FILE_TYPE_MIME_TABLE.keys():
             file_name = 'override_file_name'
+            url_encoded_file_name = quote(file_name)
```

Comment:
As you have tried to demonstrate in the tiny_example.py, could you please invest a bit more effort in writing two additional test case for the previous two demo functions.